examples/libs/walletConnection.ts

```diff
@@ -0,0 +1,29 @@
+import { AlchemyProvider } from '@ethersproject/providers';
+import { Wallet } from '@ethersproject/wallet';
+import { createStarkSigner, WalletConnection } from '@imtbl/core-sdk';
+import { requireEnvironmentVariable } from './utils';
+
+/**
+ * Generate a ethSigner/starkSigner object from a private key.
+ */
+export const generateWalletConnection = async (
```

Comment:
Should we call this class `generateSigners` or `instantiateSigners`? 

Since the Core SDK doesn't deal with wallets, we should avoid using the phrase WalletConnection

Reply:
Actually WalletConnection is the name of the interface expected by core-sdk so it is correct to call this method `generateWalletConnection`